codice/ddf
# DDF-5100 Setting Map View background color to value from theme

#### What does this PR do?
Moves the map view to a styled component and sets the background to the current theme's background color (instead of inheriting the color). 
#### Who is reviewing it? 

@Schachte 
@samuelechu 
@gordocanchola 
#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@adimka
@bdeining
@djblue

#### How should this be tested?

In intrigue, ensure that there are no changes/regressions in the map view component. The background color of the toolbar (containing `Pan to a region...`, Home, Cluster, etc) should match the current theme, as it did without this PR.
#### Any background context you want to provide?
#### What are the relevant tickets?
Fixes: #5100

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
